quiche/src/testing.rs

```diff
@@ -0,0 +1,544 @@
+// Copyright (C) 2018-2019, Cloudflare, Inc.
```

Comment:
I _think_ we have to update the license years here

Reply:
Do we need to block on that?